app/controllers/users_controller.rb

```diff
@@ -46,7 +46,7 @@ def update_password
   def export
     response.headers['Content-Type'] = 'text/csv'
     response.headers['Content-Disposition'] = 'attachment;filename=users.csv'
-    render template: 'users/export.csv.erb'
+    render template: 'users/export_csv'
```

Comment:
What need do we have to modify this here? If we have tech debt to pick up lets ticket it and lessen the size of our PRs in the future please

Reply:
Yeah, good point. I've undone this. I'll create a separate ticket as the above solution may not be the optimal one.